walk-in-interview/src/main/java/com/google/job/data/Job.java

```diff
@@ -1,18 +1,146 @@
 package com.google.job.data;
 
+import com.google.common.collect.ImmutableList;
+
+import java.util.List;
+
 /** Class for a job post. */
 public final class Job {
-    // TODO(issue/9): Add the feature to create a new job post
-    private String jobTitle;
+    private final String jobId;
+    // TODO(issue/25): merge the account stuff into job post.
+    private final JobStatus jobStatus;
+    private final String jobTitle;
+    private final Location jobLocation;
+    private final String jobDescription;
+    private final JobPayment jobPay;
+    private final List<String> requirements;
+    private final long postExpiry; // a timestamp
```

Comment:
instead of `// a timestamp` why not name the variable `postExpiryTimestamp` ?

Reply:
Fixed